server/src/project/project.service.ts

```diff
@@ -285,6 +298,8 @@ export class ProjectService {
     private readonly dynamicsWebApi: OdataService,
     private readonly config: ConfigService,
     private readonly carto: CartoService,
+    private readonly crmService: CrmService,
```

Comment:
I mean, I think it's okay to just crib the service from applicant portal, but with OdataService still there, it seems to make more sense to just copy the public methods you need. Just be aware of the lack of tests.

Reply:
Right. There are some "delete" methods in there that aren't used so far... but I figured why not crib it over in preparation for future work. hmm 🤔